crates/fluvio-storage/src/replica.rs

```diff
@@ -146,8 +146,8 @@ impl ReplicaStorage for FileReplica {
                 .into());
             }
             total_size += batch_size;
-            if batch_size > max_batch_size {
-                return Err(StorageError::BatchTooBig(max_batch_size).into());
+            if batch_size > max_request_size {
+                return Err(StorageError::BatchTooBig(max_request_size).into());
             }
         }
```

Comment:
can you add/update unit test?

Reply:
Actually, we already have integrations tests for it, I fix them. Btw, why the storage integration tests are not running on our CI?